lib/model/om/BaseDigitalObject.php

```diff
@@ -118,7 +118,7 @@ public function __get($name)
 
     try
     {
-      return call_user_func_array('QubitObject::__get', $args);
+      return parent::__get(...$args);
```

Comment:
I think this file is generated by Symfony. Was this change the result of running a `php symfony` commands?